pull_request_template.md

```diff
@@ -1,2 +1,14 @@
+_Describe the contents of your PR, how to manually test it, and provide mockups that help provide context._
 
-
+- [ ] I have manually tested the code
+- [ ] The PR name contains the ticket number
+- [ ] All of my commits contain the ticket number
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] All automated tests pass
+- [ ] I have added tests that prove my fix is effective or that my feature works
```

Comment:
Does this include creating front end tests in projects that don’t yet have them?